src/prometheus_alert_rules/host_health.rules

```diff
@@ -0,0 +1,14 @@
+groups:
+- name: HostHealth
+  rules:
+  - alert: HostAvailable
+    expr: up < 1 or absent(up)
+    for: 5m
```

Comment:
Would 5 minutes be enough for flappy alert rules? I'm wondering if that's enough for intermittent issues like networking.

Reply:
I asked the ManSol team, and this is the value they agreed upon. :)